From aadf594a267b72bb2c1a14b02a7dcc6d55c7f00d Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Mon, 27 Nov 2017 11:00:28 -0800
Subject: [PATCH] joyent/node-manta#336 node-manta affected by usdt cross-arch
 build bug Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

---
 CHANGES.md   | 6 ++++++
 package.json | 4 ++--
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 15b9d7d..89a3ce7 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,12 @@
 
 ## not yet released
 
+## 5.1.1
+
+Bring in an update to dtrace-provider by way of restify-clients.
+
+- joyent/node-manta#336 node-manta affected by usdt cross-arch build bug
+
 ## 5.1.0
 
 Minor version bump due to a backwards-compatible addition to the multipart
diff --git a/package.json b/package.json
index d819c40..138fa23 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
         "type": "git",
         "url": "git://github.com/joyent/node-manta.git"
     },
-    "version": "5.1.0",
+    "version": "5.1.1",
     "main": "./lib/index.js",
     "dependencies": {
         "assert-plus": "^1.0.0",
@@ -27,7 +27,7 @@
         "path-platform": "~0.0.1",
         "progbar": "^1.1.0",
         "readable-stream": "~1.1.9",
-        "restify-clients": "~1.4.1",
+        "restify-clients": "~1.5.2",
         "showdown": "~1.4.2",
         "smartdc-auth": "^2.4.1",
         "strsplit": "1.0.0",
-- 
2.21.0

